Doc/library/mimetypes.rst

```diff
@@ -118,6 +118,18 @@ behavior of the module.
    When *strict* is ``True`` (the default), the mapping will be added to the
    official MIME types, otherwise to the non-standard ones.
 
+.. function:: mimesniff(data)
+
+    Guess the type of given *data*.
+    *data* should be :class:`bytes` or :class:`bytearray`.
+    Return value is a tuple (type, charset).
```

Comment:
Nitpick: (type, charset) -> \`\`(type, charset)\`\`

Reply:
I think the doc should also mention the types of the returned values.
For example:
```suggestion
    Returns two string values ``(type, charset)``.
```